### PR TITLE
update WCMP2 exercise based on config creation

### PR DESCRIPTION
**Pull request by tomkralidis**

### CODE REVIEW

File: documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md [hunk 1]
```diff
@@ -49,24 +50,20 @@ more ~/wis2box-data/my-discovery-metadata.yml
...
-- `wis2box.country`: 3-letter country code in lower case
-- `wis2box.centre_id`: your centre id as defined in the previous exercise
-- `metadata.identifier`: a unique identifier consisting of `urn:x-wmo:md:[country]:[centre_id]:[dataset_name]`, where `[dataset-name]` can be any name of your choosing.  Remember this value for API validation later on in this exercise
 - `identification.title`: a human readable title describing your data
 - `identification.abstract`: a human readable description describing your data
-- `identification.dates.creation`: when the discovery metadata was created (today's date)
-- `identification.extents.spatial (bbox)`: the bounding box coordinates of your data (minimum longitude, minimum latitude, maximum longitude, maximum latitude), in decimal degrees
-- `identification.extents.temporal (begin)`: the begin and end time of your data (keeping the end time to `null` is suitable to ongoing observations)
+- `identification.extents.temporal (begin)`: the begin (`BEGIN_DATE`) and end time of your data (keeping the end time to `null` is suitable to ongoing observations)
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md:55]: Format of the BEGIN_DATE?

**Review by maaikelimper — COMMENTED** [review]
maybe add a comment that they can edit the file using WinSCP if they are not familiar with vi ?

File: documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md (hunk 1 above)
Comment (tomkralidis) [reply documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md]: Updated (`YYYY-MM-DD`).

**Comment by david-i-berry** [comment]
+1 to add comment about using WinSCP, I need to do the same in my exercises where I tell the student to edit a file.

**Comment by tomkralidis** [comment]
> maybe add a comment that they edit the discovery metadata files using WinSCP if they are not familiar with vi ?

Fixed.

File: documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md [hunk 2]
```diff
@@ -102,10 +99,13 @@ Click on your discovery metadata record and inspect the content, noting how it r
 Update the title of your discovery metadata, and re-publish:
 
 ```bash
-vi /data/wis2box/my-discovery-metadata.yml
-wis2box metadata discovery publish /data/wis2box/my-discovery-metadata.yml
+vi /data/metadata/discovery/metadata-synop.yml
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md:102]: path?

Comment (tomkralidis) [reply documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md]: Fixed (edit outside wis2box-management, but run inside).

File: documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md [hunk 3]
```diff
@@ -96,16 +96,21 @@ Ensure that your discovery metadata was also published to the broker, by looking
...
+vi ~/wis2box-data/metadata/discovery/metadata-temp.yml
 ```
 
 !!! tip
     You can also use WinSCP to connect to your instance and edit this file.
 
+Now re-publish (from inside the **wis2box-management** container):
+
+```bash
+wis2box metadata discovery publish /data/metadata/discovery/metadata-synop.yml
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md:111]: /data/wis2box/metadata/discovery/metadata-synop.yml?

File: documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md [hunk 4]
```diff
@@ -84,7 +81,7 @@ python3 wis2box-ctl.py login
 Run the following command to publish your discovery metadata:
 
 ```bash
-wis2box metadata discovery publish /data/wis2box/my-discovery-metadata.yml
+wis2box metadata discovery publish /data/metadata/discovery/metadata-synop.yml
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md:84]: wis2box metadata discovery publish /data/wis2box/metadata/discovery/metadata-synop.yml

File: documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md [hunk 5]
```diff
@@ -121,7 +126,7 @@ Feel free to update additional values and re-publishing your discovery metadata
 Run the below command to add the data to the API:
 
 ```bash
-wis2box data add-collection /data/wis2box/my-discovery-metadata.yml
+wis2box data add-collection /data/metadata/discovery/metadata-synop.yml
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/configuring-wis2-discovery-metadata.md:129]: wis2box data add-collection /data/wis2box/metadata/discovery/metadata-synop.yml

**Review by david-i-berry — APPROVED** [review]

**Pull request merged**